app/controllers/outgoing_text_messages_controller.rb

```diff
@@ -1,8 +1,30 @@
 class OutgoingTextMessagesController < ApplicationController
   include TwilioRequestable
+  process_twilio_actions only: [:update]
```

Comment:
I think `update` having such differences in how it's handled makes it a bad fit to be in the same controller as `create`. Personally, I'd like to see `update` moved out of here.